sdk/python/packages/flet-core/src/flet_core/range_slider.py

```diff
@@ -75,8 +75,8 @@ def main(page: ft.Page):
 
     def __init__(
         self,
-        start_value: [float],
-        end_value: [float],
+        start_value: float,
+        end_value: float,
```

Comment:
`Number` is coming in https://github.com/flet-dev/flet/pull/3172. We might use it everywhere once it's merged.